lms/djangoapps/labster_course_license/utils.py

```diff
@@ -105,8 +106,13 @@ def course_tree_info(store, simulations, licensed_simulations):
     Retuns information about the course's xblocks.
     """
     course_info = {}
+    invalid_simulations = []
+    validation_pattern = r'^https:\/\/[\w\d]*\.labster\.com\/[a-z]*\/[\w\d]*\/$'
     for simulation in simulations:
-        simulation_id = get_simulation_id(simulation.launch_url)
+        lti_url = simulation.launch_url
+        simulation_id = get_simulation_id(lti_url)
+        if not simulation_id or not re.search(validation_pattern, lti_url):
+            invalid_simulations.append(simulation.display_name)
```

Comment:
> invalid_simulations bad naming. Simulations are okay, wrong simulation ids :)

Not ids :) I store simulation names that have invalid lti url, renamed to `invalid_simulation_names`
@polesye maybe you have better suggestion